builtin/merged_dataset.cc

```diff
@@ -214,8 +215,10 @@ struct MergedDataset::Itl
 
         virtual const RowName & rowName(RowName & storage) const
         {
-            uint64_t hash = (*it).first;
-            return source->getRowName(RowHash(hash));
+            throw HttpReturnException(500, "FIXME");
```

Comment:
`error: returning reference to local temporary object [-Werror,-Wreturn-stack-address]`